doc/Command-Reference.md

```diff
@@ -1730,6 +1734,145 @@ This command is used to remove particular IPv4 or IPv6 BGP neighbor configuratio
+  ```
+
+- Example:
+  ```
+  admin@sonic:~$ show container
+  Usage: show container [OPTIONS] COMMAND [ARGS]...
+
+  Show container
+
+  Options:
+    -?, -h, --help  Show this message and exit.
+
+  Commands:
+    feature  Show container feature
+  ```
+
+  ```
+  admin@sonic:~$ show container feature
+  Usage: show container feature [OPTIONS] COMMAND [ARGS]...
+
+  Show container feature
+
+  Options:
+    -?, -h, --help  Show this message and exit.
+
+  Commands:
+    autorestart  Show whether the auto-restart feature for container(s) is
+                 enabled or disabled
+  ```
+
+  ```
+  admin@sonic:~$ show container feature autorestart
+  Container Name    Status
+  --------------    --------
+  database          enabled
+  syncd             enabled
+  teamd             disabled
+  dhcp_relay        enabled
+  lldp              enabled
+  pmon              enabled
+  bgp               enabled
+  swss              disabled
+  telemetry         enabled
+  sflow             enabled
+  snmp              enabled
+  radv              disabled
+  ```
+
+Optionally, you can specify a container name in order to display the auto-restart
+feature status for that container only.
+
+- Usage:
+  ```
+  show container feature autorestart database
+  ```
+
+- Example:
+  ```
+  admin@sonic:~$ show container feature autorestart database
+  Container Name    Status
+  --------------    --------
+  database          enabled
+  ```
+### Container Auto-restart config command
+
+**config container feature autorestart <container_name> <autorestart_status>**
+
+This command will configure the status of auto-restart feature for a specific container.
+
+- Usage:
+  ```
+  sudo config container feature autorestart container_name autorestart_status
+  ```
+
+- Example:
+  ```
+  admin@sonic:~$ sudo config container
+  Usage: config container [OPTIONS] COMMAND [ARGS]...
+
+    Modify configuration of container
+
+  Options:
+    -?, -h, --help  Show this message and exit.
+
+  Commands:
+    feature  Modify configuration of container features
+  ```
+
+  ```
+  admin@sonic:~$ sudo config container feature
+  Usage: config container feature [OPTIONS] COMMAND [ARGS]...
+
+    Modify configuration of container features
+
+  Options:
+    -?, -h, --help  Show this message and exit.
+
+  Commands:
+    autorestart  Configure the status of autorestart feature for specific
+                 container
+  ```
+
+  ```
+  admin@sonic:~$ sudo config container feature autorestart
+  Usage: config container feature autorestart [OPTIONS] <container_name>
+                                              <autorestart_status>
+
+  Error: Missing argument "container_name".
+  ```
+
+ ```
+  admin@sonic:~$ sudo config container feature database
+  Usage: config container feature autorestart [OPTIONS] <container_name>
+                                              <autorestart_status>
+
+  Error: Missing argument "autorestart_status".  Choose from enabled, disabled.
+  ```
```

Comment:
This output is unnecessary. Please remove.

Reply:
Removed.